src/QsCompiler/CompilationManager/ProjectManager.cs

```diff
@@ -1360,11 +1369,9 @@ public ProjectManager(Action<Exception>? exceptionLogger, Action<string, Message
         }
 
         /// <summary>
-        /// Uses FilterFiles to filter the given references binary files, and generates the corresponding errors and warnings.
-        /// Ignores any binary files that contain mscorlib.dll or a similar variant in their name.
+        /// Returns a dictionary that maps each existing dll to the Q# attributes it contains.
         /// Generates a suitable error message for each binary file that could not be loaded.
         /// Calls the given onDiagnostic action on all generated diagnostics.
-        /// Returns a dictionary that maps each existing dll to the Q# attributes it contains.
         /// </summary>
         public static ImmutableDictionary<string, References.Headers> LoadReferencedAssemblies(
```

Comment:
Does this function get used anywhere after these changes?

Reply:
Yes, it is still used by `ReloadReferencedAssembly` and by `LoadReferencedAssembliesAsync`.